qualang_tools/digital_filters/filters.py

```diff
@@ -2,19 +2,30 @@
 from typing import Tuple, List
 import numpy as np
 import scipy.signal as sig
+from warnings import warn
+from enum import Enum
+
+
+class QOP_VERSION(Enum):
+    QOP222 = {
+        "feedforward_max": 2 - 2**-16,
+        "feedback_max": 1 - 2**-20,
+        "feedforward_length": lambda feedback_len: 44 - 7 * feedback_len,
+    }
```

Comment:
Looks good! I like the approach.
I would add another enum called "None" which has no limitations